package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "legodi",
+  "version": "1.0.0",
+  "description": "Scotland Graduation Project - Digital Welcome Pack for refugees #To run the server locally use npm run dev that it will make the server work automatically when happens any change",
```

Comment:
`#To run the server locally use npm run dev that it will make the server work automatically when happens any change` this text should not be in the Despription. It should be in the README

(if you want to learn how to style README, check this https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

Reply:
also there are merge conflicts @kharazzoro

Reply:
add the text that you have in the desciption `to run ....` to the README - explain how to run the client sand server together